include/valkey/valkey.h

```diff
@@ -158,17 +162,18 @@ struct valkeySsl;
 
 #define VALKEY_OPT_NONBLOCK 0x01
 #define VALKEY_OPT_REUSEADDR 0x02
-#define VALKEY_OPT_NOAUTOFREE 0x04        /* Don't automatically free the async
-                                          * object on a connection failure, or
-                                          * other implicit conditions. Only free
-                                          * on an explicit call to disconnect()
-                                          * or free() */
-#define VALKEY_OPT_NO_PUSH_AUTOFREE 0x08  /* Don't automatically intercept and
-                                          * free RESP3 PUSH replies. */
-#define VALKEY_OPT_NOAUTOFREEREPLIES 0x10 /* Don't automatically free replies. */
-#define VALKEY_OPT_PREFER_IPV4 0x20       /* Prefer IPv4 in DNS lookups. */
-#define VALKEY_OPT_PREFER_IPV6 0x40       /* Prefer IPv6 in DNS lookups. */
-#define VALKEY_OPT_PREFER_IP_UNSPEC (VALKEY_OPT_PREFER_IPV4 | VALKEY_OPT_PREFER_IPV6)
+#define VALKEY_OPT_NOAUTOFREE                                                  \
+    0x04 /* Don't automatically free the async object on a connection failure,
```

Comment:
This change I personally dislike. We can discuss the ways to fix it but on my previous iteration I did it by moving the comment above this line.

Reply:
Moving above the line seems like a good solution.